OneSignalSDK/onesignal/in-app-messages/src/main/java/com/onesignal/inAppMessages/internal/backend/impl/InAppBackendService.kt

```diff
@@ -1,17 +1,23 @@
 package com.onesignal.inAppMessages.internal.backend.impl
 
 import com.onesignal.common.NetworkUtils
+import com.onesignal.common.consistency.impl.ConsistencyManager
 import com.onesignal.common.exceptions.BackendException
 import com.onesignal.core.internal.device.IDeviceService
 import com.onesignal.core.internal.http.IHttpClient
+import com.onesignal.core.internal.http.impl.OptionalHeaders
 import com.onesignal.debug.internal.logging.Logging
 import com.onesignal.inAppMessages.internal.InAppMessage
 import com.onesignal.inAppMessages.internal.InAppMessageContent
 import com.onesignal.inAppMessages.internal.backend.GetIAMDataResponse
 import com.onesignal.inAppMessages.internal.backend.IInAppBackendService
 import com.onesignal.inAppMessages.internal.hydrators.InAppHydrator
+import kotlinx.coroutines.delay
 import org.json.JSONObject
 
+private const val DEFAULT_RETRY_LIMIT = 3
+private const val DEFAULT_RETRY_AFTER_SECONDS = 1
```

Comment:
I don't think we should set default retry parameters on the SDK. If turbine doesn't provide these things then that should communicate to the SDK not to retry at all

Reply:
Done